02-props-state-component-architecture/react-components/todolist/src/Todo.js

```diff
@@ -0,0 +1,15 @@
+import "./Todo.css";
+import React, {Component} from 'react';
+
+class Todo extends Component {
+  render() {
+    return(
+      <div className={`Todo`}>
```

Comment:
if you have just a string, you don't need to `{}` for JavaScript, you can just do 

```jsx
<div className="Todo">
</div>
```